src/julia_init.c

```diff
@@ -0,0 +1,74 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <limits.h>
+
+#ifdef _OS_WINDOWS_
+#include <windows.h>
+#include <direct.h>
+#else
+#include <dlfcn.h>
+#endif
+
+#ifdef _MSC_VER
+JL_DLLEXPORT char *dirname(char *);
+#else
+#include <libgen.h>
+#endif
+
+// Julia headers (for initialization and gc commands)
+#include "uv.h"
+#include "julia.h"
+
+#include "julia_init.h"
+
+JULIA_DEFINE_FAST_TLS();
```

Comment:
I don't think this should be defined for libraries (see the comment in the code at https://docs.julialang.org/en/v1/manual/embedding/#High-Level-Embedding).

Reply:
What is `JULIA_DEFINE_FAST_TLS`?

Reply:
Removed in a recent commit.